commit 87d2ff308790659469d0c38e09dfb6919a95ca52 (HEAD -> debian/experimental)
Author: Reinhard Tartler <siretart@x1.int.tauware.de>
Date:   Sun May 15 21:14:30 2022 -0400

    Revert "libimage: import should not ignore configured variant if any"
    
    This reverts commit ed3dce6bfc79de229dd4ba2d2e76982d582d22ed.

Index: golang-github-containers-common/libimage/import.go
===================================================================
--- golang-github-containers-common.orig/libimage/import.go
+++ golang-github-containers-common/libimage/import.go
@@ -58,7 +58,6 @@ func (r *Runtime) Import(ctx context.Con
 		History:      history,
 		OS:           options.OS,
 		Architecture: options.Arch,
-		Variant:      options.Variant,
 	}
 
 	u, err := url.ParseRequestURI(path)
